spec.bs

```diff
@@ -3807,23 +3859,30 @@ and [=map/values=] are [=moments=] at which the cool down for the origin key exp
 </div>
 
 <div algorithm>
-  To <dfn>sample a debug report</dfn> given an [=origin=] |origin|:
+  To <dfn>sample a debug report</dfn> given an [=origin=] |origin| and a [=boolean=] |fromServer|:
 
+  Note: forDebuggingOnly reports from [=server auction response=] were downsampled on truested

```

Comment:
Typo, truested.  So the downsampling happens on the server not just for the filtered list, but also the conditional one?

Reply:
yes, it samples all. Because every url is sampled independently, so we think "sampling first on server and then pick which sampled one is going to be sent on device" is the same as "picking which ones can be sampled and then sample from them" which happens on device.